keps/sig-api-machinery/3693-admission-webhook-exclusion-rules/README.md

```diff
@@ -0,0 +1,944 @@
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+Admission Webhooks that target cluster critical resources put the admission controller backing the webhook in the critical path of cluster stability. There is currently no easy way for cluster operators to indicate to users what cluster resources are critical to cluster stability and suggest that those should be excluded from admission webhooks. Even if the cluster operator is able to make this suggestion, there is no easy way for the end user to exclude these resources (or any other resources of interest to them) from the admission webhooks. 
+
+Adding `exclusionRules` to `ValidatingWebhookConfiguration` and `MutatingWebhookConfiguration` would enable users to have a more expressive API for their admission webhooks and would provide a path for cluster-operators to suggest cluster critical resources to exclude from admission webhooks that is simple, configurable and transparent to the user.
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
```

Comment:
nit: you can delete these comments. They're just part of the template to provide some instruction.